test/shortcuts.spec.js

```diff
@@ -293,19 +298,25 @@ describe('Shortcuts component', () => {
       },
     }
     )
+
     baseContext.shortcuts.setKeymap(editedKeymap)
 
-    simulant.fire(node, 'keydown', { keyCode: space })
+    // sometimes node loose focus, so I added this timeout to fix it
+    setTimeout(() => {
```

Comment:
Is this timeout really necessary? Why did it work before without it?

Reply:
It's good question. Currently I don't have time to deep debugging. I will back to this PR a bit later and will provide more info (or solution without monkey-patching that must be). If you do not do this faster than me sure :)